dotCMS/src/main/java/com/dotcms/rest/DeletePPQueueElementsByIdentifierForm.java

```diff
@@ -0,0 +1,34 @@
+package com.dotcms.rest;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import java.util.List;
+
+@JsonDeserialize(builder = DeletePPQueueElementsByIdentifierForm.Builder.class)
+public class DeletePPQueueElementsByIdentifierForm {
+
+    private final List<String> identifiers;
+
+    private DeletePPQueueElementsByIdentifierForm(final Builder builder) {
+
+        this.identifiers = builder.identifiers;
+    }
+
+    public List<String> getIdentifiers() {
+
+        return this.identifiers;
+    }
+
+    public static final class Builder {
+        @JsonProperty(required = true) private List<String> identifiers;
```

Comment:
Why does it need this Jackson annotation?

Reply:
Because it's a required param